engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/CreateProcessInstanceProcessor.java

```diff
@@ -118,6 +121,17 @@ public boolean onCommand(
     return true;
   }
 
+  @Override
+  public ProcessingError tryHandleError(
+      final TypedRecord<ProcessInstanceCreationRecord> typedCommand, final Throwable error) {
+    if (error instanceof EventSubscriptionException exception) {
```

Comment:
:question: Is this always true? Or is it only true when we are in a "token move" context?

Reply:
I've clarified it in a comment, and git commit message